source/scpp/src/xdr/Stellar-types.h

```diff
@@ -9,8 +9,9 @@
 
 namespace stellar {
 
-using Hash = xdr::opaque_array<32>;
+using Hash = xdr::opaque_array<64>;
 using uint256 = xdr::opaque_array<32>;
+using uint512 = xdr::opaque_array<64>;
```

Comment:
Can you add a comment above those types, explaining that they were modified on purpose ?